gradle.properties

```diff
@@ -0,0 +1,3 @@
+org.gradle.daemon=true
+org.gradle.configureondemand=true
```

Comment:
Deamon and configuration on demand caused problems before, i think we should keep them false